packages/gatsby/src/redux/actions/public.js

```diff
@@ -1424,9 +1424,13 @@ actions.createServerVisitedPage = (chunkName: string) => {
  * @param {Object} manifest Manifest data
  * @param {string} manifest.manifestId An id which ties the revision unique state of this manifest to the unique revision state of a data source.
  * @param {Object} manifest.node The Gatsyby node to tie the manifestId to. See the "createNode" action for more information about the node object details.
+ * @param {string} manifest.updatedAt (optional) The time in which the node was last updated
+ * @param {number} manifest.daysSinceLastUpdate (optional) Choose the days since a node was last updated (ie create manifests for nodes that were last updated up to 30 days)
```

Comment:
We should mention that this is in UTC - probably the api should include that in the name too. `maxDaysOldUTC`

Reply:
`maxDaysOld` is just a number of a days, why would that need to be UTC? Also the `updatedAt` can technically be anything that we can instantiate a date from, so it can be whatever format the data source needs it to be.

Reply:
Oh oops! I meant `updatedAtUTC` 🤦 and the reason is we're comparing using `Date.now()` which is the number of seconds since Jan 1, 1970 in UTC time. Using a local timezone to compare here will be innacurate so the API always requires entering time in UTC